tests-legacy/PrestaShopBundle/Utils/DatabaseCreator.php

```diff
@@ -49,6 +51,7 @@ public static function createTestDB()
         require_once __DIR__ . '/../../../install-dev/init.php';
 
         $install = new Install();
+        Shop::setContext(Shop::CONTEXT_SHOP, 1);
```

Comment:
This doesn't seem to be enough the integration tests are still failing

Reply:
and phpunit as well it seems